### DISCUSSION TITLE
Possible to capitalize first letter in the name of the group

### DISCUSSION THREAD

**Discussion by Lumca** [scalar/scalar#1271] [Q&A] [answered]
Hello!

There could be a way to  capitalize first letter in the name of the group

![image](https://github.com/scalar/scalar/assets/4989415/d0581571-05fb-4fd9-ade0-bfc87f006f47)
As it is in stripe
![image](https://github.com/scalar/scalar/assets/4989415/74e5589c-2608-4a2b-81cd-c1460201efc9)

I tried to use custom css but no luck

It is uppercase or lowercase only:
```
.sidebar-heading {
  text-transform: lowercase;
}

.sidebar-heading sidebar-group-item__folder::first-letter {
  text-transform: uppercase;
}
```

thanks!

**Comment by antlio** [comment]
hey @Lumca thanks for the feedback! that makes sense, would it work for you if the case is by default not style and just reflecting how it's written in the definition?

**Reply by Lumca** [reply]
Hi @antlio , that would be great, thanks!

**Reply by antlio** [reply, marked as answer]
hi there, we just merged and release #1793 and #1828, it is still uppercase by default however you will be now able to overload using the variable:
```css
--scalar-tag-text-transform
```
